packages/terra-application-navigation/src/header/CompactHeader.module.scss

```diff
@@ -0,0 +1,132 @@
+:local {
+  .compact-header {
+    align-content: center;
+    align-items: center; // TODO THEME
```

Comment:
Should this be a themable variable I'm guessing based on the TODO comment.

Reply:
I'm thinking all the colors in this file should likely be themable. I believe the consumer theme uses dark colors for navigation icons on a light background where clinical theme uses light/white colors on a dark background for navigation icons.

Reply:
Updated the colors. For the time being I think I'll leave the alignment unthemed. In dev-site we originally had this start aligned, but that was prop driven.

Reply:
Updated.